admin-dev/themes/new-theme/js/theme.js

```diff
@@ -47,6 +42,13 @@ import './product-page/index';
 import './translation-page/index';
 
 import Header from './header.js';
+import initDatePickers from './app/utils/datepicker';
+
+const $ = global.$;
 
 new NavBar();
 new Header();
+
+$(() => {
+  initDatePickers();
```

Comment:
This probably should be done only in pages containing datepickers, shouldn't it?

Reply:
We need to check that adding it globally doesn't conflict with datepickers management in product page.

Reply:
@mickaelandrieu It's a fix because no datepickers working on product page =)

Reply:
Ok but we will have to do it eventually when we'll split the files per page sometime in the future